### ISSUE TITLE
UI: Job submission message contains text "undefined" instead of number of submitted jobs

### ISSUE THREAD

**Issue by bernt-matthias** [galaxyproject/galaxy#14410]
**Describe the bug**

When submitting a tool on a collection the message shown to the user is:

```
Executed tool XYZ and successfully added undefined jobs to the queue. 
```

I guess that instead of the the word "_undefined_" the number of submitted jobs should be displayed.

**Galaxy Version and/or server at which you observed the bug**

22.01 

**To Reproduce**

Execute a tool on a collection (e.g. on usegalaxy.eu)

**Expected behavior**

Show number of jobs.

**Screenshots**

![Screenshot from 2022-08-08 10-44-40](https://user-images.githubusercontent.com/25689525/183377714-9dba14b3-1636-49ab-b23d-d974b1d145e9.png)

**Additional context**

**Comment by dannon** [comment]
@bernt-matthias Thanks for the report, should be fixed in #14420

**Comment by nsoranzo** [comment]
Fixed in https://github.com/galaxyproject/galaxy/pull/14420 .
